src/components/Modals/PublishCollectionModalWithOracle/PublishCollectionModal.tsx

```diff
@@ -80,10 +80,15 @@ export default class PublishCollectionModal extends React.PureComponent<Props, S
       <>
         <ModalNavigation title={t('publish_collection_modal_with_oracle.title')} onClose={onClose} />
         <Modal.Content className="first-step">
-          {isFetchingItems || isFetchingRarities || !refRarity ? (
+          {isFetchingItems || isFetchingRarities ? (
             <div className="loader-wrapper">
               <Loader size="big" active={isFetchingItems || isFetchingRarities} />
             </div>
+          ) : !!itemError || !refRarity ? (
```

Comment:
Will `itemError` only hold rarities related errors? 
I'm asking because if there's an error but `refRarity` has value, it will show the rarities fetch error.

Reply:
rarities is inside the item sagas so it includes both item and rarities errors